### PR TITLE
Develop

### PR DESCRIPTION
**Pull request by RomanPerepelkin**
https://RomanPerepelkin.github.io/react_redux-list-of-posts/

### CODE REVIEW

File: src/features/users.ts [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+
+type UsersState = {
+  users: User[];
+};
+
+const initialState: UsersState = {
+  users: [],
+};
+
+export const loadUser = createAsyncThunk('users/fetch', async () => {
```
Comment (TarasHoliuk) [inline src/features/users.ts:14]:
```suggestion
export const loadUsers = createAsyncThunk('users/fetch', async () => {
```

File: src/components/PostDetails.tsx [hunk 2]
```diff
@@ -1,118 +1,130 @@
...
-    // eslint-disable-next-line max-len
-    setComments(currentComments =>
-      currentComments.filter(comment => comment.id !== commentId),
-    );
-
-    await commentsApi.deleteComment(commentId);
-  };
+  }, [selectedPost, dispatch]);
+
+  // const [comments, setComments] = useState<Comment[]>([]);
```
Comment (TarasHoliuk) [inline src/components/PostDetails.tsx:27]: remove comments

File: src/components/PostDetails.tsx [hunk 3]
```diff
@@ -1,118 +1,130 @@
...
 import { Loader } from './Loader';
 import { NewCommentForm } from './NewCommentForm';
+import * as commentsActions from '../features/comments';
 
-import * as commentsApi from '../api/comments';
+// import * as commentsApi from '../api/comments';
 
-import { Post } from '../types/Post';
-import { Comment, CommentData } from '../types/Comment';
+// import { Post } from '../types/Post';
```
Comment (TarasHoliuk) [inline src/components/PostDetails.tsx:8]: remove comments

fix all similar cases

**Review by TarasHoliuk — CHANGES_REQUESTED** [review]
Great job!

Code looks good, but you left a lot of commented code - let's add a cleanup and remove it

**Review by volodymyr-soltys97 — CHANGES_REQUESTED** [review]
Good job 👍 
Let's improve your code
1. You need to resolve merge conflicts 
image

2. The comment cannot be only spaces 
image

3. Add a correct type to input of email 
image

**Comment by RomanPerepelkin** [comment]
Please tell me how to fix conflict

**Review by anastasiiavorobiova — CHANGES_REQUESTED** [review]
Please, ask questions in the chat or during the QA sessions. It's much more effective. Mentors can see your comments just once during the review

**Pull request closed without merging**